apps/src/templates/progress/ProgressLessonContent.jsx

```diff
@@ -46,14 +46,18 @@ export default class ProgressLessonContent extends React.Component {
       ));
     }
 
-    return (
-      <div>
-        <div style={styles.summary}>
-          <SafeMarkdown markdown={description || ''} />
+    if (description) {
+      return (
+        <div>
+          <div style={styles.summary}>
+            <SafeMarkdown markdown={description} />
+          </div>
+          {bubbles}
         </div>
-        {bubbles}
-      </div>
-    );
+      );
+    } else {
+      return <div>{bubbles}</div>;
+    }
```

Comment:
Thanks for the attention to details! 

Wondering if it works to always render `<div>{bubbles}</div>` and only conditionally render the description to eliminate the `else` branch? 
Something like: 
```
return (
        <div>
          {description && (
               <div style={styles.summary}>
                  <SafeMarkdown markdown={description} />
              </div>
          )}
            <div> {bubbles} </div>
      </div>
    );

Reply:
Excellent- thank you!